src/main/java/seedu/address/model/house/Level.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.house;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+
+/**
+ * Represents a House's level.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLevel(String)}
+ */
+public class Level {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Level should only contain numbers, and it should only be at most 2 digits long";
+    public static final String VALIDATION_REGEX = "\\d{1,2}";
+    public final String value;
+
+    /**
+     * Constructs a {@code Level}.
+     *
+     * @param level A valid level.
+     */
+    public Level(String level) {
+        requireNonNull(level);
+        checkArgument(isValidLevel(level), MESSAGE_CONSTRAINTS);
+        value = level;
+    }
+
+    /**
+     * Returns true if a given string is a valid level.
+     */
+    public static boolean isValidLevel(String test) {
+        return test.matches(VALIDATION_REGEX);
```

Comment:
Looks good, but I think I will give the same comment as the "Block" class. Is it possible that users can key in '0' or '00' for level? I am not too sure whether there is 0 level for HDB/Condo. However, if you done your research and there is 0 level, then ignore my comment on this.